pkg/server/nodemanager/node_controller.go

```diff
@@ -43,7 +43,7 @@ import (
 // non-const timeouts were endorsed by Mitchell Hashimoto
 var (
 	//	BootTimeout         time.Duration = 300 * time.Second
-	BootTimeout         time.Duration = 15 * time.Minute
+	BootTimeout         time.Duration = 20 * time.Minute
```

Comment:
We probably should have different timeouts per instance types.

This is fine for now.